_i18n/fr/index.md

```diff
@@ -0,0 +1,190 @@
+                                <div class="col">
+                                    <p><a href="{{site.baseurl}}/blog/">Plus de Nouvelles</a></p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- end right one-third block-->
+            </div>
+        </section>
+
+        <!-- END TWO THIRDS & ONE THIRD BLOCKS ROW -->
+
+        <!-- 3 ONE-THIRD BLOCKS ROW -->
+
+        <section class="container">
+            <div class="row">
+                <!-- left one-third block-->
+                <div class="left one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>Moneropedia</h2>
+                                <p class="main-resources">Souhaitez-vous rechercher les significations des termes et concepts utilisés dans Monero ? Vous trouverez ici un guide alphabétique des termes et leurs significations des projets Monero et Kovri.</p>
+                                <p><a href="{{site.baseurl}}/resources/moneropedia/" class="btn-link btn-auto">Lire Moneropedia</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- end left one-third block-->
+
+                <!-- center one-third block-->
+                <div class="center one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>User Guides</h2>
+                                <p class="main-resources">Les guides pas à pas de tous les aspects de Monero sont séparés par catégories et couvrent tout, de la création d'un portefeuille au soutient du réseau, et même comment modifier ce site.</p>
+                                <p><a href="{{site.baseurl}}/resources/user-guides/" class="btn-link btn-auto">Lire les guides utilisateur</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- end center one-third block-->
+
+                <!-- right one-third block-->
+                <div class="right one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>FAQ</h2>
+                                <p class="main-resources">Nous avons entendu beaucoup de questions au cours des années et avons compilé, pour votre commodité, une FAQ complète et variée. Ne vous inquiétez pas, si vos questions ne sont pas ici, vous pouvez toujours demander à la communauté.</p>
```

Comment:
This text is too long. It ends outside the box

Reply:
Rephrased to 196 characters.